### PR TITLE
adding the ReadMe Markdown File

### PR DESCRIPTION
**Pull request by javyW**
Javyen Ware's Copy of the ReadMe Markdown file

### CODE REVIEW

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Would like you to include something that speaks to how co-workers can best engage with you. Please update to include these details. Otherwise, well done!

**Review by javyW — COMMENTED** [review]
I changed a part of my markdown file to include how best co-workers can engage with you.

File: lesson_00/JavyenWare/RM.markdown
Comment (anthonydmays) [inline lesson_00/JavyenWare/RM.markdown:1]: This file should be named README.md, I missed this on the original review.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_00/JavyenWare/RM.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## My Favorite Things
+
+
+
+- Favorite Games: Yokai Watch 3 for 3ds, Titanfall 2, 13 Sentinels Aegis Rim, or Terraria
+- Favorite Movies: Interstellar or Wreck it Ralph
+- Favorite Cereals: Fruit pebbles, Honey Nut Cheerios, or Cinnamon Toast Crunch
+
+<img src="img/CoolTerrariaPics.jpeg" alt="Cool Terraria Pics" width="200" height="200">
```
[the remark this answers is not in the document]
Comment (javyW) [reply lesson_00/JavyenWare/RM.markdown]: I Fixed the image. How I fixed it was that you have to copy and paste the image url from the original image into the src tag.

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**